test/test_force.js

```diff
@@ -369,10 +370,14 @@ function createCompileApp(tmpDir, os, actualAppType, templateRepoUri, pluginRepo
 }
 
 function buildForiOS(target, workspaceDir, appName) {
```

Comment:
Generated app for the SPM template only has a xcode project no workspace (in the other templates, the workspace is created by cocoapod).